packages/manager/modules/support/src/tickets/new-ticket/feedback/feedback.html

```diff
@@ -0,0 +1,5 @@
+<a
+    data-ng-href="{{ $ctrl.url }}"
+    rel="noopener"
+    target="_blank"
+    data-translate="support_tickets_new_feedback"></a>
```

Comment:
No icon or aria description to indicate it opens in a new window ?

Reply:
Shameful indeed !